packages/monorepo-releaser/release.config.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * @fileoverview semantic-release-wildcards semantic-release configuration
+ */
+const configMonorepo = require('./').configMonorepo;
+
```

Comment:
Don't we need the tagFormat property so it releases it as the correct package name that's scoped?

Reply:
only if these packages are being name-scoped. Both module names are available in npmjs, so no need to scope